e2e/suites/provider/cases/common/common.go

```diff
@@ -657,3 +658,45 @@ func DecodingPolicySync(f *framework.Framework) (string, func(*framework.TestCas
 		}
 	}
 }
+
+func CollectingOrphanSecrets(f *framework.Framework) (string, func(*framework.TestCase)) {
+	return "[common] should delete orphaned secrets with Owner creation policy", func(tc *framework.TestCase) {
+		secretKey1 := fmt.Sprintf("%s-%s", f.Namespace.Name, "foo")
+		secretValue := "value"
```

Comment:
I think the value must be json, as vault doesn't allow you to store a simple string.